app.js

```diff
@@ -1,83 +1,67 @@
 $(document).ready(function () {
   console.log("js up and running!!");
 
-
+// INDENTATION is off throughout this whole file.
+// Everything between curly braces should be indented.
+// $("#cocktail-by-name-btn").on("click".. is inside document.ready's callback, 
+// so it needs to be indented in a level 
 $("#cocktail-by-name-btn").on("click", function() {
-  // to clear my section each time so I can keep searching w/o refresh
   let resultClear = $(".drink-details");
   resultClear.html("");
   
   let whatISearched = $("#cocktail-by-name").val();
   $.ajax({
+    // indentation is off here as well
         method: "GET",
         url: "https://www.thecocktaildb.com/api/json/v1/1/search.php?s=" + whatISearched,
         success: onSuccess,
         error: function onError(drinks) {
-          console.log("api broke, yo!");}
+          console.log("api broke for search, yo!");}
       });
 
   function onSuccess (drinks) {
-    console.log(drinks);
```

Comment:
Do not leave unnecessary `console.log`s in your code – they are good for testing things out as you go, but they are not necessary for the program to work properly. Clean out all `console.log`s before submitting.